src/components/Button/Button.js

```diff
@@ -61,31 +60,23 @@ const Button = ({
     return null;
   })();
 
-  const button = (
-    <button
-      {...other}
-      {...commonProps}
-      disabled={disabled}
-      type={type}
-      ref={other.inputref}>
-      {children}
-      {buttonImage}
-    </button>
-  );
-
-  const anchor = (
-    <a
+  let Component = 'button';
+  if (render) {
+    Component = render;
+  } else if (href) {
+    Component = 'a';
+  }
+  return (
+    <Component
```

Comment:
Cool - Seems that we need to cover `disabled` and `type` for `<button>`.

Reply:
fixed (sorry for the delay, i accidentally changed `inputref` to `inputRef` and was confused why all the tests were failing...)